python/paddle/tensor/random.py

```diff
@@ -104,6 +104,48 @@ def bernoulli(x, name=None):
         return out
 
 
+@dygraph_only
+def bernoulli_(x, p=0.5, name=None):
+    """
+    This is the inplace version of api ``bernoulli``, which returns a Tensor filled
+    with random values sampled from a bernoulli distribution. The output Tensor will
+    be inplaced with input ``x``. Please refer to :ref:`api_tensor_bernoulli`.
+
+    Args:
+        x(Tensor): The input tensor to be filled with random values.
+        p (float|Tensor, optional): The success probability parameter of the output Tensor's bernoulli distribution.
```

Comment:
Should it be explained and checked whether `p` must be broadcast to `x`

Reply:
We could support the scenarios. Let me update the code.